### PR TITLE
Voron2.4_exhaust_filter_Duct

### PR DESCRIPTION
**Pull request by 2718564960**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
./Edwardlining/Voron2.4_exhaust_filter_Duct)
#### How was it tested? 
print it and use it by myself. You will see the picture in the file
#### Any background context you want to provide?
No

### CODE REVIEW

**Comment by 2718564960** [comment]
Voron2.4_exhaust_filter_Duct 
I want to contribute something . It's my first contribute , if there is something wrong please tell me . Thank you

**Comment by MapleLeafMakers** [comment]
@2718564960 this PR has gotten a bit old, and needs to have the latest upstream changes pulled in.   You may find it easier to simply backup your mod files, delete your fork, re-create it, and open a new PR.  If you go this route, you should consider doing your work on a separate branch, rather than `master`.

File: printer_mods/README.md [hunk 1]
```diff
@@ -339,4 +339,4 @@ like so:
 | | [V0 Skirt set](./Zen3D/V0_Skirt)| SSR, PowerSwitch, Nameplate and color change printing  | :heavy_check_mark: |:x: |:x: |:x:|
 | | [V0 1515 T-nut](./Zen3D/V0_1515_T-nut)| M3 square and hex nuts for Voron0 1515 frame | :heavy_check_mark: |:x: |:x: |:x:|
 | zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts which auto-align (LDO frames only) | :heavy_check_mark: | :x: | :x: | :x: |
-
+| Edwardlining | [V2.4 exhaust](./Edwardlining/Voron2.4_exhaust_filter_Duct) | Voron2.4 exhaust filter Duct | :x: |:grey_question: |:heavy_check_mark:| :x: | :x:|
```
Comment (MapleLeafMakers) [inline printer_mods/README.md:342]: Mods should be listed in alphabetical order (by username). This should come after `eddie`

**Comment by MapleLeafMakers** [comment]
CAD Source files must be provided in either STEP or SCAD format.

**Comment by FHeilmann** [comment]
Since there was no response from the submitter, I'm closing this PR for now. If you'd still like to submit this mod, feel free to reach out to me (Haribro) on discord for assistance.

**Pull request closed without merging**